### ISSUE TITLE
[CMS COLLAB CYCLE INTAKE] (Public Websites, Home page, Hub list)

### ISSUE THREAD

**Issue by dsasser** [department-of-veterans-affairs/va.gov-cms#12044]
### What type of request are you making?

- [X] Update to an existing CMS feature (including product iterations, updating DS components or page layouts and nav changes)
- [ ] New CMS feature (New content types, DS components, workflows or products)
- [ ] Content audit (either one time audits or resuable tools)
- [X] Change Management review (for any visible changes to the CMS, workflow or content model)
- [ ] Other

### Please choose the team that is requesting the Collab Cycle kickoff.

Public Websites

### When did/will you start working on this product/feature?

We are in investigation phase only

### When will your feature need to launch?

Soon after new homepage is released.

### Are you doing research with VA.gov users or editors?

No

### Will your work involve changes to...?

CMS content model

### Please add a link to your product outline issue.

Does not exist yet

### Additional Information

In the updated homepage design, we are adding a new item to the hub list that isn't likely to be coming from a Benefit Hub Landing Page, since it will require a link out to an external OPIA controlled resource: 
![Screen Shot 2022-12-12 at 2 05 17 PM](https://user-images.githubusercontent.com/85581471/207165209-aca11656-88cc-4a57-828e-e3d7982c78b7.png)

![screencapture-va-gov-2022-12-20-12_26_16](https://user-images.githubusercontent.com/221539/208760243-bf762276-7298-497e-a799-f8a977716534.png)

See [this spike](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11370) for additional context.

Dave C has challenged us to come up with a way to allow a single source of truth to power the hubs list. Below is our proposal

## Changes being proposed:
Do away with the exiting method of using an entityqueue to power the benefit hub list on the home page (see screenshots below). Entityqueue's are limited to a single entity type, so mixing content (BHLP) with blocks (for instance) is not possible. Additionally, adding a second content type will just make managing hub list items more difficult.

More details can be found in [the spike](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11370#:~:text=isn%27t%20a%20BHLP%3F%3A-,Replace%20with%20menu,-Click%20to%20expand).

I have mocked up the idea on my local, including FE changes in content-build. This results in a seamless transition from entityqueue to menu:

Current homepage using a menu rather than entityqueue:
<img width="1493" alt="Screen Shot 2022-12-20 at 11 33 48 AM" src="https://user-images.githubusercontent.com/221539/208759713-19aacd44-1e16-4084-a302-d35c980f0e8f.png">

Current v2 homepage using a menu rather than entityqueue:
<img width="1554" alt="Screen Shot 2022-12-20 at 11 33 56 AM" src="https://user-images.githubusercontent.com/221539/208759751-33a09745-1b0c-4066-bc3d-6a9f64547a1b.png">

### Notify the Sitewide CMS team of this ticket in the cms-support Slack channel.

- [X] I acknowledge that I must notify `#cms-support` after submitting this issue.

**Comment by EWashb** [comment]
If I'm reading this correctly, @dsasser, this won't drastically change the editorial experience. Or please correct me if I'm wrong.

**Comment by dsasser** [comment]
> If I'm reading this correctly, @dsasser, this won't drastically change the editorial experience. Or please correct me if I'm wrong.

Well, it introduces a *new* editorial experience for home page hub list items, which affects the current process of using a Benefits Hub Landing Page + EntityQueue to build the items. I don't know if that is drastic or not, but it is a change.

Further, it introduces questions about the fields on the BHLP which are leveraged by the hub list:

<img width="1130" alt="Screen Shot 2022-12-22 at 10 51 17 AM" src="https://user-images.githubusercontent.com/221539/209206696-14251e94-2cc1-46b2-962c-27af1c41bd29.png">

**Comment by EWashb** [comment]
Good points @dsasser. Correct me if I'm wrong, but this seems to be more of a high priority than the other collab cycle request? cc: @jilladams

**Comment by jilladams** [comment]
We have 3 open requests, and I am pretty sure this is the priority order: 

1. https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9912 -- highest priority but also our real thinking is blocked on Sitewide Content team next steps.
2. https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12044
3. https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12062

**Comment by jilladams** [comment]
**Additional context:** 
* [Original slack thread](https://dsva.slack.com/archives/C03NSH4SLQY/p1671125891943039) with Swirt & Daniel where we discussed options and arrived at proposal to use a menu
* Driving force for this change = adding a VA Department Info link to the hub list, that points out to a Wordpress site, e.g. in [Homepage Round 3 designs](https://www.sketch.com/s/3aa40506-4be2-46cc-876b-93f1a9f3a857/p/EAD305BC-3082-4F81-ADEC-C2486F7B6BEF/canvas). 
  * That would corrupt the current Benefit Hub Landing Page model for the hub list managed via entityqueue
* Proposal: Manage hub list content in a menu moving forward (Icon, title, description, link)
* Menu will include: 
  * Hub list icon - select list that will not re-use any existing icon lists, and will include VA Dept Info
  * Hub label
  * Hub teaser text
  * Public Websites team will include draft copy for CMS UI in the build

**Next touchpoint:**
* Mid-point review not needed because this is using Core Drupal
* Implementation review, at PR time:
  * Laura: Accessibility review, and will flag any concerns about the editor experience
  * Steve: Code review

**Future work:**
* Existing H2: "Explore VA Benefits & Health Care" may no longer apply, with VA Dept Info in the mix. We may want to consider a change. (Not ticketed.)
* Existing Homepage Listing fields on Benefit Hub Landing Pages can be deprecated when the new home page hard launches. (#11921)
  * Possible epic: refactor Benefit Hub Landing Pages to work out how it should change after the fact. (Should we establish Drupal SSOT for Hub data), not ticketed. 
* Permissions: Menus are currently locked to Drupal Admins. That's fine for now. 
  * editors = Sitewide Content, and they confirmed they rarely make changes. 
  * We can open menu permissions to a specified role if needed, will need to confirm scope of what's needed first.

**Comment by jilladams** [comment]
Shipped.
